h/templates/admin/groups.html.jinja2

```diff
@@ -21,7 +21,15 @@
       <tbody>
         {% for group in results %}
           <tr>
-            <td>{{ group.name }}</td>
+            <td>
+              {% if group.type in ('open', 'restricted') %}
+              {% set edit_url = request.route_url('admin_groups_edit', pubid=group.pubid) %}
+              <a href="{{ edit_url }}">{{ group.name }}</a>
+              {% else %}
+              {# The group edit view does not currently support editing private groups. #}
+              {{ group.name }}
+              {% endif %}
```

Comment:
Just out of interest why isn't editing private groups supported? Is there any reason why it wouldn't just work?

Reply:
The schema for the form doesn't have "private" as a supported group type yet. Otherwise I think it would work.